tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/AnalyticSummaryPopup.xaml

```diff
@@ -0,0 +1,125 @@
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+
+        <StackPanel Padding="15">
+            <TextBlock
+                x:Uid="AnalyticSummary_Title"
+                Style="{StaticResource SubtitleTextBlockStyle}"
+                HorizontalAlignment="Left"
+                Margin="16,10,0,0" />
+            <TextBlock
+                x:Uid="AnalyticSummary_Description"
+                HorizontalAlignment="Left"
+                Margin="16,10,0,0" />
+
+            <!-- Filters & sorting -->
+            <Grid Margin="15 40 15 5">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition Width="*"/>
+                </Grid.ColumnDefinitions>
+
+                <TextBlock Grid.Column="0" x:Uid="AnalyticSummary_SortBy_Label" Margin="5 5 5 5" />
+
+                <ComboBox
+                    Grid.Column="1"
+                    x:Name="SortComboBox"
+                    Margin="5 0 5 0"
+                    PlaceholderText="Sort by"
+                    SelectedValue=""
+                    SelectedIndex="{x:Bind ViewModel.SortComboBoxIndex, Mode=TwoWay}">
+                    <ComboBoxItem x:Uid="ComboBox_SortBy_Process" />
+                    <ComboBoxItem x:Uid="ComboBox_SortBy_Type" />
+                    <ComboBoxItem x:Uid="ComboBox_SortBy_CpuAboveThreshold" />
+                    <i:Interaction.Behaviors>
+                        <ic:EventTriggerBehavior EventName="SelectionChanged">
+                            <ic:InvokeCommandAction Command="{x:Bind ViewModel.SortProcessesComboBoxChangedCommand, Mode=OneWay}"
+                                    CommandParameter="{x:Bind ((ComboBoxItem)SortComboBox.SelectedItem).Content, Mode=OneWay}" />
+                        </ic:EventTriggerBehavior>
+                    </i:Interaction.Behaviors>
+                </ComboBox>
+
+                <TextBox
+                    Grid.Column="2"
+                    x:Uid="AnalyticSummary_FilterProcesses_PlaceholderText"
+                    x:Name="FilterTextBox"
+                    TextChanged="FilterTextBox_TextChanged"
+                    Margin="50 0 0 0" />
+            </Grid>
+        </StackPanel>
+
+        <ScrollViewer VerticalScrollBarVisibility="Auto" Grid.Row="1" Padding="10 0 10 0">
+            <StackPanel Padding="15 0 15 15">
+                <local:SimpleTableControl x:Name="processListControl" ItemsSource="{x:Bind ViewModel.ProcessDatasAd}" />
+            </StackPanel>
+        </ScrollViewer>
+
+        <Grid Grid.Row="2" Background="{StaticResource SmokeFillColorDefaultBrush}">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*"/>
+                <ColumnDefinition Width="*"/>
+            </Grid.ColumnDefinitions>
+
+            <Button
```

Comment:
A content dialog rolls with a close and save button.  Why make your own?

Reply:
I'm getting lost in the theming of the built in buttons.

    this.PrimaryButtonStyle = (Style)Application.Current.RequestedTheme.GetType().GetProperty("AccentButtonStyle").GetValue(Application.Current.RequestedTheme);

Reply:
I'm looking at the ContentDialog in the WinUI 3 Gallery, and it seems like the PrimaryButton is Accented by default? Unless that's not what you want. (It's available in the store, also https://github.com/microsoft/WinUI-Gallery)

Reply:
https://github.com/microsoft/devhome/blob/main/tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs#L419 is the hack I used to make the primary button accented

Reply:
It's really weird to me that the behavior you're seeing here doesn't match the Gallery.